### PR TITLE
Edits to documentation in book_source

### PR DESCRIPTION
**Pull request by Aariq**
## Description

As I go through the PEcAn bookdown site, I've been making edits where I find typos, missing links, or outdated instructions.  It's likely to be a lot of small commits as I go through the book, so let me know if you'd like me to "squash" them and I can figure out how to do that.

## Motivation and Context
Updating documentation.

## Review Time Estimate

- [ ] Immediately
- [ ] Within one week
- [x] When possible

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue) 
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] My name is in the list of CITATION.cff
- [x] I have updated the CHANGELOG.md. 
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes. (not applicable)
- [x] All new and existing tests passed. (some tests failing, but unrelated to these changes)

### CODE REVIEW

File: book_source/01_introduction/01_project_overview.Rmd [hunk 1]
```diff
@@ -50,15 +50,15 @@ Any opinions, findings, and conclusions or recommendations expressed in this mat
...
+* LeBauer, D., Kooper, R., Mulrooney, P., Rohde, S., Wang, D., Long, S. P. and Dietze, M. C. (2017), betydb: a yield, trait, and ecosystem service database applied to second-generation bioenergy feedstock production. GCB Bioenergy. [doi:10.1111/gcbb.12420](https://doi.org/10.1111/gcbb.12420)
+* Rogers A, BE Medlyn, J Dukes, G Bonan, S von Caemmerer, MC Dietze, J Kattge, ADB Leakey, LM Mercado, U Niinemets, IC Prentice, SP Serbin, S Sitch, DA Way, S Zaehle. 2017. "A Roadmap for Improving the Representation of Photosynthesis in Earth System Models" New Phytologist 213(1):22-42 [doi:10.1111/nph.14283](https://doi.org/10.1111/nph.14283)
+* Shiklomanov. A, MC Dietze, T Viskari, PA Townsend, SP Serbin. 2016 "Quantifying the influences of spectral resolution on uncertainty in leaf trait estimates through a Bayesian approach to RTM inversion" Remote Sensing of the Environment 183: 226-238. [doi:10.1016/j.rse.2016.05.023](https://doi.org/10.1016/j.rse.2016.05.023)
+* Viskari et al. 2015 Model-data assimilation of multiple phenological observations to constrain and forecast leaf area index. Ecological Applications 25(2): 546-558. [doi:10.1890/14-0497.1](https://doi.org/10.1890/14-0497.1)
```
Comment (infotroph) [inline book_source/01_introduction/01_project_overview.Rmd:61]: 👍

**Comment by Aariq** [comment]
I can't get the book to build with `bookdown::serve_book()` so can't check my fixes.

File: book_source/02_demos_tutorials_workflows/01_install_pecan.Rmd [hunk 2]
```diff
@@ -152,11 +152,12 @@ This will not go into much detail about about how to use Docker -- for more deta
     - PEcAn web interface (running models) -- http://localhost:8000/pecan/ (NOTE: The trailing backslash is necessary.)
     - PEcAn documentation and home page -- http://localhost:8000/
     - BETY web interface -- http://localhost:8000/bety/
-    - File browser (minio) -- http://localhost:8000/minio/
+    - File browser (minio) -- http://localhost:8000/minio/ <!-- ERS: Doesn't work on docker installation -->
```
Comment (dlebauer) [inline book_source/02_demos_tutorials_workflows/01_install_pecan.Rmd:155]: please update w/ instructions from @robkooper on Slack https://pecanproject.slack.com/archives/C8RD9BRLK/p1655147681727739 which should work with fixes in #2936

**Review by dlebauer — COMMENTED** [review]
Thanks for these updates! A great place to start with your first PR to PEcAn!

Please update the Docker section once you get that working (see rob's slack msg).

Will wait to approve and merge until you are done with revisions.

**Comment by dlebauer** [comment]
> I can't get the book to build with `bookdown::serve_book()` so can't check my fixes unfortunately.

That is okay. GitHub actions should trigger and rebuild (either automatically, or manually by an admin for your first PR)

**Comment by Aariq** [comment]
Might be good if @robkooper checked that I got all the URLs right.

**Review by mdietze — APPROVED** [review]

**Comment by Aariq** [comment]
Ok, I added instructions for the r-universe installation.  I kept the existing [native install instructions](https://pecanproject.github.io/pecan-documentation/develop/install-native.html), but I could also delete those and instead link to https://pecanproject.github.io/pecan-documentation/develop/osinstall.html

**Comment by Aariq** [comment]
I updated ALL the documentation to use the new docker endpoints at http://pecan.localhost/.  This change isn't reflected in tests, shell scripts, and other code though, so I'll open a separate issue.

File: book_source/02_demos_tutorials_workflows/01_install_pecan.Rmd [hunk 3]
```diff
@@ -149,16 +153,30 @@ This will not go into much detail about about how to use Docker -- for more deta
...
-    - PEcAn web interface (running models) -- http://localhost:8000/pecan/ (NOTE: The trailing backslash is necessary.)
-    - PEcAn documentation and home page -- http://localhost:8000/
-    - BETY web interface -- http://localhost:8000/bety/
-    - File browser (minio) -- http://localhost:8000/minio/
-    - RabbitMQ management console (for managing queued processes) -- http://localhost:8000/rabbitmq/
-    - Traefik, webserver showing maps from URLs onto their respective containers -- http://localhost:8000/traefik/
-    - Monitor, service that monitors models and shows all models that are online as well as how many instances are online and the number of jobs waiting. The output is in JSON -- http://localhost:8000/monitor/
-    
+    - PEcAn documentation and home page -- http://pecan.localhost
+    - PEcAn web interface (running models) -- http://pecan.localhost/pecan/ (NOTE: The trailing backslash is necessary.)
```
Comment (robkooper) [inline book_source/02_demos_tutorials_workflows/01_install_pecan.Rmd:157]:
```suggestion
    - PEcAn web interface (running models) -- http://pecan.localhost/pecan/ (NOTE: The trailing slash is necessary.)
```

File: book_source/03_topical_pages/94_docker/03_architecture.Rmd [hunk 4]
```diff
@@ -234,7 +234,7 @@ Our configuration is as follows:
 yaml::write_yaml(dc_yaml$services["rabbitmq"], stdout())
 ```
 
-Note that the `traefik.frontend.rule` indicates that browsing to http://localhost:8000/rabbitmq/ leads to the RabbitMQ management console.
+Note that the `traefik.frontend.rule` indicates that browsing to http://pecan.localhost/rabbitmq/ leads to the RabbitMQ management console.
```
Comment (robkooper) [inline book_source/03_topical_pages/94_docker/03_architecture.Rmd:237]:
```suggestion
Note that the `traefik.http.routers.rabbitmq.rule` indicates that browsing to http://rabbitmq.pecan.localhost/ leads to the RabbitMQ management console.
```

File: book_source/03_topical_pages/94_docker/03_architecture.Rmd [hunk 5]
```diff
@@ -167,7 +167,7 @@ Our current configuration is as follows:
 yaml::write_yaml(dc_yaml$services["minio"], stdout())
 ```
 
-The Minio interface is accessible by browsing to `localhost:8000/minio/`.
+The Minio interface is accessible by browsing to `pecan.localhost/minio/`.
```
Comment (robkooper) [inline book_source/03_topical_pages/94_docker/03_architecture.Rmd:170]:
```suggestion
The Minio interface is accessible by browsing to `minio.pecan.localhost`.
```

Unfortunately we can no longer use the pecan.localhost/minio/ :(

File: book_source/03_topical_pages/94_docker/05_building_images.Rmd [hunk 6]
```diff
@@ -103,7 +103,7 @@ NOTE: All commands assume you are working from the PEcAn source code root direct
    
 3. To update to the latest local code, run `./scripts/docker_rebuild.sh`.
    Under the hood, this uses `curl` to post a RabbitMQ message to a running Docker instance.
-   By default, the scripts assumes that username and password are both `guest` and that the RabbitMQ URL is `http://localhost:8000/rabbitmq`.
+   By default, the scripts assumes that username and password are both `guest` and that the RabbitMQ URL is `http://pecan.localhost/rabbitmq`.
```
Comment (robkooper) [inline book_source/03_topical_pages/94_docker/05_building_images.Rmd:106]:
```suggestion
   By default, the scripts assumes that username and password are both `guest` and that the RabbitMQ URL is `http://rabbitmq.pecan.localhost/`.
```

probably need to update the shell script in that case.

**Review by robkooper — CHANGES_REQUESTED** [review]

File: book_source/03_topical_pages/94_docker/05_building_images.Rmd (hunk 6 above)
Comment (Aariq) [reply book_source/03_topical_pages/94_docker/05_building_images.Rmd:106]: Yeah, there's a separate issue for finishing this switch in code #2955

**Comment by robkooper** [comment]
should squash and merge

**Pull request merged**